query-engine/driver-adapters/src/types.rs

```diff
@@ -126,129 +126,149 @@ impl JSResultSet {
     }
 }
 
-#[cfg_attr(not(target_arch = "wasm32"), napi_derive::napi)]
-#[cfg_attr(target_arch = "wasm32", derive(Clone, Copy, Deserialize_repr))]
-#[repr(u8)]
-#[derive(Debug)]
-pub enum ColumnType {
-    // [PLANETSCALE_TYPE] (MYSQL_TYPE) -> [TypeScript example]
+macro_rules! js_column_type {
```

Comment:
Little comment on the syntax of that macro would be nice, i.e. what's on the left side, what's on the right, what's in parenthesis. It's not too difficult to figure that out from the code, but still would be nice, especially since most of the names on the right and left are identical.